### DISCUSSION TITLE
Helpful Links For Filecoin Miners

### DISCUSSION THREAD

**Discussion by TippyFlitsUK** [filecoin-project/community#143] [Mining]
**`Please comment below to request additions or edits.`**

**Filecoin Official**
[Filecoin - Home](https://filecoin.io/)
[Filecoin - Blog](https://filecoin.io/blog/)
[Filecoin - Slack](https://filecoin.io/slack)
[Filecoin - Discussions](https://github.com/filecoin-project/lotus/discussions)
[Filecoin - Docs](https://docs.filecoin.io/)
[Filecoin - Security & Bug Bounty](https://security.filecoin.io/)
[Filecoin - GitHub](https://github.com/filecoin-project)
[Filecoin - GitHub Activity Board](https://github.com/orgs/filecoin-project/projects/11)
[Filecoin - Systems Status](https://filecoin.statuspage.io/)
[Filecoin - Network Status](https://network.filecoin.io/)
[Filecoin - Spec](https://spec.filecoin.io/)

**Explorers**
[Filfox - Filecoin browser](https://filfox.info/en)
[Filscout - Filecoin Explorer](https://filscout.io/en)
[Filscan - Filecoin Explorer](https://filscan.io/#/home)
[Filecoin Chain Stats - Grafana](https://stats.filecoin.io/d/z6FtI92Zz/filecoin-chain-stats?orgId=1&refresh=30s&from=now-30m&to=now&kiosk)

**Running Lotus**
[Managing MPOOL - MPOOL function and methods of administration - ribasushi - PLFD](https://filecoinproject.slack.com/archives/C0179RNEMU4/p1607269156412600?thread_ts=1606987775.021900&cid=C0179RNEMU4)
[Using Control Addresses - Minimise message traffic jams with separate process control addresses -  Patrick - Factor8 Solutions](https://filecoinproject.slack.com/archives/CEGN061C5/p1608729108487600?thread_ts=1608727648.485800&cid=CEGN061C5)
[Storage Path Weight Explained - Magik6k](https://filecoinproject.slack.com/archives/C0179RNEMU4/p1601378224411200?thread_ts=1601378070.410800&cid=C0179RNEMU4)
[Message Method - What the numbers mean](https://github.com/filecoin-project/specs-actors/blob/e195950ba98adb8ce362030356bf4a3809b7ec77/actors/builtin/methods.go#L78-L102)

**Build**
[Textile - A network of applications, connected through interoperable data, where data is owned by the users.](https://textile.io/)
[Pinata - Add Files To IPFS Effortlessly](https://pinata.cloud/)
[IPFS Powers the Distributed Web](https://ipfs.io/)
[ProtoSchool - Filecoin Course](https://proto.school/course/filecoin)

**Projects**
[Slate - Decentralized Storage](https://slate.host/)
[Project Staring - A Framework For Data Integrity](https://www.starlinglab.org/)
[Filehive - Filecoin Marketplace](https://beta.filehive.app/datasets/trending)

**Miner Reputation & Deal Marketplaces**
[FilSwan](https://www.filswan.com/#/dashboard)
[FilRep - Filecoin Reputation System](https://filrep.io/?sortBy=score&order=desc)
[ConsenSys Codefi - Storage](https://www.storage.codefi.network/)

**Metrics**
[Spacegap - Dashboard for Proof of Space](https://spacegap.github.io/)
[Fgas - Real-time Filecoin Mining Gas Queries](https://fgas.io/index)
[file.app - Filecoin miner performance, activity, and data](https://file.app/)

**Various Recommended Hardware**
[Canada - PartPicker](https://pcpartpicker.com/user/tperson/saved/#view=H2BskL)
[mining-hardware-config-testnet-v3.pdf - OUTDATED](https://filecoin.io/vintage/mining-hardware-config-testnet-v3.pdf)
[Top Mining Pool 6Block Shares its Filecoin Mining Hardware Specs](https://medium.com/6block/top-mining-pool-6block-shares-its-filecoin-mining-hardware-specs-205fbb270ac1)

**Config**
[How do I add swap after system installation? - Ask Ubuntu](https://askubuntu.com/questions/33697/how-do-i-add-swap-after-system-installation)
[Permanently Set ULIMIT Value](https://tecadmin.net/increase-open-files-limit-ubuntu/)
[Configuring ZFS on Ubuntu 20.04 - LinuxConfig.org](https://linuxconfig.org/configuring-zfs-on-ubuntu-20-04)
[MultiAddr GitHub - Publish IP](https://github.com/multiformats/multiaddr)
[Open Port Check Tool -- Verify Port Forwarding on Your Router](https://canyouseeme.org/)
[Install Nvidia GPU Drivers Using NVIDIA CLI Installer - stuberman](https://filecoinproject.slack.com/archives/CEGN061C5/p1611092973223600?thread_ts=1610141465.497500&cid=CEGN061C5)

**Dashboards**
[Farcaster Dashboard - s0nik42](https://github.com/s0nik42/lotus-farcaster)

**Calculators**
[FileCoin Calc - Google Sheets - Eliovp](https://docs.google.com/spreadsheets/d/1g7DdnvdCmMkKqNS4um2sfWVm9kIsaqfVrYzZ9F_ayaQ/edit#gid=0)

**Comment by JDLH** [comment]
Typo: the link text "Project Staring - A Framework For Data Integrity" should read "Starling", with an "l". Evidence: the link destination URL has this "l" in it.
